### PR TITLE
[TIMOB-17693] Support new AlertDialog properties

### PR DESCRIPTION
**Pull request by hansemannn**
_JIRA_: https://jira.appcelerator.org/browse/TIMOB-17693

### CODE REVIEW

File: iphone/Classes/TiUIAlertDialogProxy.m [hunk 1]
```diff
@@ -35,7 +35,9 @@ -(NSMutableDictionary*)langConversionTable
 			@"title",@"titleid",
 			@"ok",@"okid",
 			@"message",@"messageid",
-			nil];
+            @"loginPlaceholder", @"passwordPlaceholder",
```
Comment (cheekiatng) [inline iphone/Classes/TiUIAlertDialogProxy.m]: indentation here should be consistent with the above. use tab.

Comment (cheekiatng) [inline iphone/Classes/TiUIAlertDialogProxy.m]: What is the purpose of adding these 3 new objects here? This is a dictionary with Objects and Keys, there's no corresponding keys for these objects.

**Comment by cheekiatng** [comment]
FT done. Please address comments.

**Comment by hansemannn** [comment]
@cheekiatng Fixed the conversation table

**Comment by cheekiatng** [comment]
APPROVED.

**Pull request merged**